src/e2e-test/features/bigquery/source/BigQuerySourceError.feature

```diff
@@ -55,3 +55,24 @@ Feature: BigQuery source - Validate BigQuery source plugin error scenarios
     Then Enter BigQuery source property table name
     Then Enter BigQuery property temporary bucket name "bqInvalidTemporaryBucket"
     Then Verify the BigQuery validation error message for invalid property "bucket"
+
+  @BQ_SOURCE_TEST
+  Scenario Outline:To verify error message when unsupported format is provided in Partition Start date and Partition end Date
+    Given Open Datafusion Project to configure pipeline
+    When Expand Plugin group in the LHS plugins list: "Source"
+    When Select plugin: "BigQuery" from the plugins list as: "Source"
+    Then Navigate to the properties page of plugin: "BigQuery"
+    Then Replace input plugin property: "project" with value: "projectId"
+    Then Replace input plugin property: "dataset" with value: "dataset"
+    Then Replace input plugin property: "table" with value: "bqSourceTable"
+    Then Click on the Get Schema button
+    Then Enter BigQuery source properties partitionFrom and partitionTo
+    Then Validate BigQuery source incorrect property error for Partition Start date "<property>" value "<value>"
```

Comment:
@Praveena2607 I don't think the internal review comments have been addressed. Please address all the review comments provided earlier in this PR as well. Ref https://github.com/Praveena2607/google-cloud/pull/1

Reply:
As discussed over call, Please resolve all the discussed points and mentioned in internal PR. To use the latest framework steps in all scenarios, for entering the values in property, for validating inline error messages, for selecting output schema macro.